### PR TITLE
Fixes #16862: ncf fails to load on solaris

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/16862

### CODE REVIEW

File: tree/10_ncf_internals/list-compatible-inputs [hunk 1]
```diff
@@ -12,7 +12,7 @@ then
 fi
 
 # add a default path since it is emptied by cfengine, and busybox for android
-PATH="/sbin:/bin:/usr/sbin:/usr/bin:/usr/locan/sbin:/usr/local/bin:/system/xbin:${PATH}"
+PATH="/usr/gnu/bin:usr/gnu/x86_64-pc-solaris2.11/bin:/sbin:/bin:/usr/sbin:/usr/bin:/usr/locan/sbin:/usr/local/bin:/system/xbin:${PATH}"
```
Comment (amousset) [inline tree/10_ncf_internals/list-compatible-inputs]: missing leading / in second path?
also what is 2.11?

Comment (ncharles) [reply tree/10_ncf_internals/list-compatible-inputs]: what is /usr/locan/bin ? (not in the change, but in the line)

Comment (amousset) [reply tree/10_ncf_internals/list-compatible-inputs]: good spot @ncharles

Comment (peckpeck) [reply tree/10_ncf_internals/list-compatible-inputs]: erf, lets say useless

**Comment by peckpeck** [comment]
PR updated with a new commit

**Pull request merged**